diode-proto/diode/v1/reconciler.proto

```diff
@@ -61,6 +61,15 @@ enum State {
   NO_CHANGES = 4;
 }
 
+// Log Stats
+message Stats {
+  optional int32 total = 1;
+  optional int32 new = 2;
+  optional int32 reconciled = 3;
+  optional int32 failed = 4;
+  optional int32 no_changes = 5;
```

Comment:
What's reason behind use of `optional` here? Without it we would default to 0 which is fine.

Reply:
Would be great to have similar corresponding values as in https://github.com/netboxlabs/diode/blob/develop/diode-proto/diode/v1/reconciler.proto#L56-L62

```
enum State {
  UNSPECIFIED = 0;
  NEW = 1;
  RECONCILED = 2;
  FAILED = 3;
  NO_CHANGES = 4;
}
```

Reply:
don't get the enum comparation

Reply:
Ah, mea culpa: conflated enum iota values with field numbers here 🙈

Reply:
done, removed optional